taskrunners/cleaner/Dockerfile

```diff
@@ -4,3 +4,5 @@ COPY requirements.txt requirements.txt
 COPY setup_requirements.sh setup_requirements.sh
```

Comment:
I think setup_requirements.sh needs to be added to this directory

Reply:
Removed the COPY step. setup_requirements was copied from somewhere else. It's not used for anything.